test/bigtest/network/factories/contact.js

```diff
@@ -0,0 +1,18 @@
+import { Factory, faker } from '@bigtest/mirage';
+
+export default Factory.extend({
+  user: () => faker.random.uuid(),
+  personal: () => {
+    return {
+      firstName: faker.random.words(),
```

Comment:
[faker has `name.firstName` and `name.lastNamer` generators.](https://github.com/Marak/faker.js)